postings/views.py

```diff
@@ -100,4 +100,48 @@ def get(self, request):
         
         except AttributeError:
             return JsonResponse({'message':'AttributeError'}, status=400)
+
+class CommentView(View):
+    @login_decorator
+    def post(self,request,posting_id):
+        try:
+            data    = json.loads(request.body)
+
+            posting= Posting.objects.get(id=posting_id)
+            reply   = data['reply'] 
+            user    = request.user
+            
+            Comment.objects.create( 
+                reply   = reply,
+                user    = user,
+                posting = posting
+            )
+
+            return JsonResponse({"message" : "SUCCESS"}, status=201)
+
+        except KeyError:
+             return JsonResponse({"message" : "KEY_ERROR"}, status=401)
+
+        except Posting.DoesNotExist:
+            return JsonResponse({"message": "INVALID_POSTING"}, status=400)     
+    
+    def get(self,request,posting_id):
+        try:
+        
+            comments = Posting.objects.get(id=posting_id).comment_set.values('reply','user__name')
+            
+            results= [comment for comment in comments]
+
+            return JsonResponse({"message":"SUCCESS", "results" : results}, status=201)
+        
+        except Comment.DoesNotExist:
```

Comment:
`Posting.objects.get`을 사용하셨으니 반환되는 인스턴스가 없는 경우 `Comment.DoesNotExist`가 아니라 `Posting.DoesNotExist`가 raise 되겠죠?
```suggestion
        except Posting.DoesNotExist:
```